packages/components/filters/filters-list/src/filters-list.tsx

```diff
@@ -26,21 +29,15 @@ export type TFiltersConfiguration = {
   isDisabled?: boolean;
 };
 
-export type TFilterGroup = {
+export type TFilterGroupConfiguration = {
   key: string;
   label: ReactNode;
 };
 
-export type TAddFilterSelectOption = {
-  label: string;
-  value: string;
-  isDisabled: boolean;
-};
-
 export type TFiltersListProps = {
-  filters: TFiltersConfiguration[];
-  filterGroups?: TFilterGroup[];
-  appliedFilters: TAppliedFilterValue[];
+  filters: TFilterConfiguration[];
+  filterGroups?: TFilterGroupConfiguration[];
+  appliedFilters: TAppliedFilter[];
```

Comment:
Nit: the prop is called "appliedFilters" but the type is "AppliedFilter". Can we either add an s or not to both? Doesn't have to be updated here, but just going forward.

Reply:
I figured that `appliedFilters` is the plural of an array of singular `appliedFilter` objects.  But I have a pretty low care amount here so I can change it if necessary